liveblocks.config.ts

```
@@ -0,0 +1,61 @@
+  username?: string;
+  image?: string;
+  content: string;
+  createdAt: number;
+};
+declare global {
+  interface Liveblocks {
+    // Each user's Presence, for useMyPresence, useOthers, etc.
+    Presence: {
+      // Example, real-time cursor coordinates
+      // cursor: { x: number; y: number };
+    };
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)